modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyDLObjectsMVCActionCommand.java

```diff
@@ -81,13 +81,15 @@ protected void doProcessAction(
 			(UploadException)actionRequest.getAttribute(
 				WebKeys.UPLOAD_EXCEPTION);
 
+		int itemsCopied = 0;
+
 		if (uploadException != null) {
 			errorMessages.add(
 				_getUploadExceptionErrorMessage(uploadException, themeDisplay));
 		}
 		else {
 			try {
-				_copyDLObjects(actionRequest, errorMessages, themeDisplay);
+				itemsCopied = _copyDLObjects(actionRequest, errorMessages, themeDisplay);
```

Comment:
instead of calculate on the function
can we do not extract it as (pseudocode)

what do you think?
```
_newFunction(actionRequest, errorMessages){
long[] dlObjectIds = ParamUtil.getLongValues(
			actionRequest, "dlObjectIds");
total = dlObjectIds.length
return total-errorMessages.size();
}
```

Reply:
Makes sense, I just did it in my last commit.